renderdoc/CMakeLists.txt

```diff
@@ -383,6 +383,7 @@ set(data
     data/glsl/minmaxtile.comp
     data/glsl/quadresolve.frag
     data/glsl/quadwrite.frag
+    data/glsl/pixelhistory.frag
```

Comment:
We might need more shaders than just this one for pixel history by the end, so I think a better name would be `pixelhistory_primid.frag`.

Reply:
Renamed.